arch/risc-v/src/mpfs/Kconfig

```diff
@@ -637,6 +637,11 @@ config MPFS_CORESPI_IRQNUM_OFFSET
 	default 1
 	depends on MPFS_CORESPI
 
+config MPFS_CORESPI_DMA
+	bool "CoreSPI FPGA DMA support"
+	default y
```

Comment:
Probably n is better initially, until FPGA is in place

Reply:
It's fine, we don't need to merge this before FPGA is available